typescript/syntax/Expressions.sdf3

```diff
@@ -16,6 +16,6 @@ context-free syntax
   PrimaryExpression.Call = <<PrimaryExpression>(<{PrimaryExpression ","}*>)>
   PrimaryExpression.FunctionExpression = FunctionDeclaration
   PrimaryExpression.PropertyLookup = <<PrimaryExpression>.<ID>>
+  PrimaryExpression.New = <new <PrimaryExpression>(<{PrimaryExpression ","}*>)>
```

Comment:
according to the syntax, in: `new X(...)`, 
`X` can be any primary expression. I guess this is to support: `new Outer.Inner(...)` or something similar, but I do not really see why this is represented by `PrimaryExpression` since pretty much any true expression is illegal in this position.